packages/core/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
These diffs look wrong.

Reply:
I think this looks okay now. I'm not sure what had the `fuse.js` dep before tho 🤔 `npm ls fuse.js` only shows this package.